constantine/ethereum_verkle_primitives.nim

```diff
@@ -0,0 +1,35 @@
+# Constantine
+# Copyright (c) 2018-2019    Status Research & Development GmbH
+# Copyright (c) 2020-Present Mamy André-Ratsimbazafy
+# Licensed and distributed under either of
+#   * MIT license (license terms in the root directory or at http://opensource.org/licenses/MIT).
+#   * Apache v2 license (license terms in the root directory or at http://www.apache.org/licenses/LICENSE-2.0).
+# at your option. This file may not be copied, modified, or distributed except according to those terms.
+
+## ############################################################
+##
+##              Verkle Trie primitives for Ethereum
+##
+## ############################################################
+
+import
+  ./math/config/[type_ff, curves],
+  ./math/arithmetic,
+  ./math/elliptic/ec_twistededwards_projective,
+  ./math/io/[io_bigints, io_fields],
+  ./curves_primitives
+  
+
+func mapToBaseField*(p: ECP_TwEdwards_Prj[Fp[Banderwagon]]): Fp[Banderwagon] =
```

Comment:
don't return a result, this is likely to cause slowness without NRVO (Named Return Value Optimization) / copy-elision: https://github.com/nim-lang/Nim/issues/19357

Reply:
Ohh right 
Will resolve this